.github/ISSUE_TEMPLATE/issue-report.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Issue report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**For Confluent Enterprise Support customers, we would strongly advise you to open a Support ticket which will be addressed within your Support contract SLA at ```https://support.confluent.io```**
+
+**Describe the issue**
+A clear and concise description of what the issue is.
+
+**To Reproduce**
+Steps to reproduce the behaviour:
+
+**Expected behaviour**
+A clear and concise description of what you expected to happen.
+
+**Logs**
```

Comment:
maybe add logs of the component if possible. I usually just use the `journalctl -efu confluent-server` to get logs, but i know you just go to /var/log